test/controllers/bill-runs.controller.test.js

```diff
@@ -101,4 +101,23 @@ describe('Bill Runs controller', () => {
       })
     })
   })
+
+  describe('GET /bill-runs/{id}/review', () => {
+    let options
+
+    beforeEach(async () => {
+      options = {
+        method: 'POST',
+        url: '/charge-elements/time-limited'
```

Comment:
I think you've got a copy & paste error! I'm guessing this should be

```suggestion
        url: '/bill-runs/b6e71397-c2fb-4c7f-a2c7-1e2e1cc14e58/review'
```